TESTS/events/queue/main.cpp

```diff
@@ -22,7 +22,7 @@
 
 using namespace utest::v1;
 
-#define TEST_EQUEUE_SIZE 1024
+#define TEST_EQUEUE_SIZE (18*EVENTS_EVENT_SIZE)
```

Comment:
If I am not familiar with event_size, why this is 18 ? The commit neither any code/comment here provides that information how this number was selected

Reply:
There is no strict answer.
1024 was to large to make a test passed on NUCLEO_F070RB + GCC_ARM compiler so the value was reduced below 1024.
I thought it will be more readable if expressed in `EVENTS_EVENT_SIZE` 
same as is in `EventQueue` constructor

```
/** EVENTS_QUEUE_SIZE
 *  Default size of buffer for events
 */
#define EVENTS_QUEUE_SIZE (32*EVENTS_EVENT_SIZE)

EventQueue(unsigned size=EVENTS_QUEUE_SIZE, unsigned char *buffer=NULL);
```

Reply:
OK that explains it, we are testing maximum of 18 events.

What I was after that this is a fix, and commit does not explain how it is fixing the issue (reducing event queue size to 18 events, as test does not require more, or something similar).